BlitzBoard/Features/HomeScreenViewModel.swift

```diff
@@ -43,8 +43,8 @@ class HomeScreenViewModel {
             case .success(let teamStandings):
                 self?.leagueTables = teamStandings
                 self?.delegate?.reloadView()
+                // #Warning("Will remove print statement in the future ")
```

Comment:
don't put the //. it mkes it a comment and not a warning

Reply:
When I remove the // I get an error saying "No macro named 'Warning'"

Reply:
I removed the print statement, just cleaning up code